frame/staking/src/lib.rs

```diff
@@ -1211,9 +1221,10 @@ decl_module! {
 					log!(warn, "💸 Estimating next session change failed.");
 				}
 			}
-
-			// weight
-			50_000
+			add_weight(3, 0, 0);
```

Comment:
```suggestion
			// Because of call into `estimate_next_new_session`
			add_weight(3, 0, 0);
```

Reply:
well ok but then you have to also add 3 read for:
```rust
				Self::era_election_status().is_closed() &&
				// either current session final based on the plan, or we're forcing.
				(Self::is_current_session_final() || Self::will_era_be_forced())
```

Reply:
@kianenigma I added a comment, but this read counter is for what @thiolliere pointed out. as far as I understand, estimate_next_new_session doesn't do storage reads?

Reply:
I think estimate_next_new_session do 3 storage read: https://github.com/paritytech/substrate/pull/5964#discussion_r424317932

Reply:
fixed